cadc-vosi/src/main/java/ca/nrc/cadc/vosi/AvailabilityPlugin.java

```diff
@@ -96,4 +96,10 @@
      * @param state requested state
      */
     public void setState(String state);
+    
+    /**
+     * Returns HTTP status 200 OK to indicate that a service is alive.
+     * This method is to be implemented by all services.
+     */
+    public void heartBeat();
```

Comment:
Again, 'heartbeat' would be better. 

I wonder if it should be made clear how implementations indicate that the heartbeat check failed.  Something like:

public boolean heartbeat()  (return true or false)
public void heartbeat() throws RuntimeException  (thrown exception means failed heartbeat)...

Reply:
This also needs more clear javadoc saying that this method should be a very lightweight test that could be called every few seconds. The javadoc should not say anything about HTTP codes - just that this is a very lightweight test that the service is (probably) working.

Reply:
Changed method name and signature. Also updated javadoc.